docs/book/src/user/quick-start.md

```diff
@@ -143,12 +143,12 @@ export AWS_SESSION_TOKEN=<session-token> # If you are using Multi-Factor Auth.
 # The clusterawsadm utility takes the credentials that you set as environment
 # variables and uses them to create a CloudFormation stack in your AWS account
 # with the correct IAM resources.
-clusterawsadm alpha bootstrap create-stack
+clusterawsadm bootstrap iam create-cloudformation-stack
```

Comment:
Since these will only be available with a future release, this should also probably add a minimum version requirement in the download instructions for `clusterawsadm` above.

Reply:
+1 
probably worth to keep documented somewhere the procedure for older releases and link it (for  older releases see ...)

Reply:
Added both.